### ISSUE TITLE
CCL consistently segfaults (through CLASS) at NERSC

### ISSUE THREAD

**Issue by damonge** [LSSTDESC/CCL#539]
If you install CCL at NERSC following the instructions in https://confluence.slac.stanford.edu/display/~heather/Building+CCL+at+NERSC (which are quite outdated), it segfaults whenever it tries to compute the CLASS power spectrum.

I've managed to track this down to somewhere within `evolver_ndf15`, in `tools/evolver_ndf15`, but then I ran out of stamina.

**Comment by damonge** [comment]
Alright, further on this: the issue seems to be caused by the `-ffast-math` compiler flag used by CLASS by default, coupled with the version of gcc that NERSC's `cc` uses.

I'll investigate whether these options would be easily tuneable through cmake, but more savvy people (@EiffL ?) may have other suggestions...

**Comment by damonge** [comment]
Currently addressed by PR #540 . Will close when merged.

**Comment by rainwoodman** [comment]
I am curious -- were you using the gcc 7.3 (module list gcc will show)?

**Comment by EiffL** [comment]
I looked at the PR, it looks good, but I'm very confused as to why this is happening, it used to work fine at NERSC. I tried to log in and try myself today, only to be reminded about this MFA business that I have had no patience for so far... I have a bunch of deadlines coming up, not sure I'll reset my NERSC access in a least a week time-scale.
I would usually recommend pushing this as an issue upstream (to the CLASS repo), but if this issue is a show stopper for you, I'm ok with merging your PR.
